src/sum.js

```diff
@@ -6,12 +6,19 @@
  * @returns {number} The sum
  */
 function sum(array) {
-  // This is your job. :)
+  let count = 0
+
+  for(let element in array) {
```

Comment:
If you want to loop over indexes, use `for(let i = 0;i < array.length; i++) { ... }`

If you want to loop over values, use `for (let num of array) { ... }`